iloop_to_model/iloop_to_model.py

```diff
@@ -147,6 +147,7 @@ def extract_measurements_for_phase(scalars_for_samples):
 MEASUREMENTS = 'measurements'
 REACTIONS = 'reactions-knockout'
 MODEL = 'model'
+GROWTH_RATE = 'growth-rate'
```

Comment:
I'd prefer underscores or camelcase (although not very Python-y). In js you can't access property names containing `-` with the dot notation, making it less convenient. WDYT? Also mentioning @kvikshaug

Reply:
Note that this doesn't reach JS; the response from venom **is** camelcased in the end. This is used in the request to the `model` service.
However I'm all for standardizing on camelcase as part of a larger task.